vignettes/faq.Rmd

```diff
@@ -67,3 +65,7 @@ tfrmt(
   print_to_gt(es_data) %>%
   tab_options(container.width = 1000)
 ```
+
+### How do I output my table?
+
+To output your table, use `tfrmt`'s `print_to_gt` and `print_mock_gt` functions, which create `gt` table objects. We recommend leveraging the export capabilities provided by the `gt` package to save and share your table. For more information on exporting, see [the gt package documentation](https://gt.rstudio.com/reference/gtsave.html).
```

Comment:
It would be good to clarify the difference between viewing the table in R via the print functions and outputting to a document. A few changes to make:
- Add a sentence up front stating that in order to share your table with others, you likely want to output it to a document.
- Clarify that tfrmt's print functions create a gt object, which can be viewed in R. 
- Clarify that tfrmt does not offer functionality to save the table to a document, instead this is handled by gt
- Good to mention the variety of document types supported by gt

Reply:
not sure what i've done to make a test fail @bzkrouse? its something to do with 
Failed to get R 4.4.2: Failed to install qpdf?